github_issue_summarization/distributed/tfjob.yaml

```diff
@@ -1,69 +0,0 @@
----
-apiVersion: "kubeflow.org/v1alpha2"
```

Comment:
any reason we're deleting it? It would still make sense to keep it for educational purpose for people not familiar with sonnet

Reply:
The TFEstimator code doesn't work; so running the code with YAML will just produce errors. So I think keeping it right now is just a source of confusion. If/when we fix the code to work with TF.Estimator we can restore it.

Reply:
But only change needed is command and image, so it'd be very easy to make yaml work and I think it's more readable than ksonnet. Why not fix yaml rather than deleting it?

Reply:
If you'd like to do that in a follow on PR; I'm open to it. I'd like to get the example back to a healthy state and having less files is helpful.